components.yaml

```diff
@@ -35,6 +35,12 @@ components:
     branch: master
     update-policy: static
     metadata: ""
+  multusV3:
+    url: https://github.com/k8snetworkplumbingwg/multus-cni
+    commit: f4c0adf54c99d7395d30a050a8d29b674b99d700
+    branch: master
+    update-policy: tagged
+    metadata: v3.9.1
```

Comment:
Why 3.9.1 instead of [3.9.3](https://github.com/k8snetworkplumbingwg/multus-cni/releases/tag/v3.9.3) (latest) ?

Reply:
I just reverted to what CNAO used to deploy before the transition to Multus v4 (taken from v0.80.0)